### PR TITLE
basic ui changes apply

### PR DESCRIPTION
**Pull request by rajasekarcbt**
i have added images, css, js, and plugins— unique changes

i have added a header, footer, home page and related css files.

home page includes
1. testimonial and related css files
2. venue and related files

I have included pdf file below after testing my local server.

header-new
homepage-section1
homepage-section2
homepage-section3
homepage-section4
footer-new

### CODE REVIEW

File: backend/frontend/public/assets/plugins/owl-carousel/owl.video.play.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
+<html><head>
+<title>404 Not Found</title>
+</head><body>
+<h1>Not Found</h1>
+<p>The requested URL was not found on this server.</p>
+<hr>
+<address>Apache/2.4.52 (Ubuntu) Server at dreamsports.dreamstechnologies.com Port 443</address>
+</body></html>
```
Comment (murucbt) [inline backend/frontend/public/assets/plugins/owl-carousel/owl.video.play.html]: Format this and update the content as discussed.

File: backend/frontend/public/index.html [hunk 2]
```diff
@@ -11,16 +11,27 @@
...
+  <link rel="stylesheet" href="%PUBLIC_URL%/assets/plugins/select2/css/select2.min.css" />
+  <link rel="stylesheet" href="%PUBLIC_URL%/assets/css/bootstrap-datetimepicker.min.css">
+  <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
   <script src="https://use.fontawesome.com/releases/v6.3.0/js/all.js" crossorigin="anonymous"></script>
-
+  <script src="%PUBLIC_URL%/assets/plugins/owl-carousel/owl.carousel.min.js" type="3ed837b2bc5bdc76df64b3e0-text/javascript"></script>
+  <script src="%PUBLIC_URL%/assets/plugins/fancybox/jquery.fancybox.min.js" type="3ed837b2bc5bdc76df64b3e0-text/javascript"></script>
+  <script src="%PUBLIC_URL%/assets/js/script.js" type="b1bc3b7a2a5c1b7cc97da585-text/javascript"></script>
+  <script src="%PUBLIC_URL%/assets/plugins/theia-sticky-sidebar/theia-sticky-sidebar.js" type="76371501ea8e4a4f5e72eced-text/javascript"></script>
+
```
Comment (murucbt) [inline backend/frontend/public/index.html]: Try to arrange it asc order.

File: backend/frontend/src/App.js [hunk 3]
```diff
@@ -8,40 +8,41 @@ import BookingInfoScreen from "./screens/BookingInfoScreen";
...
-import UpdateprofileScreen from './screens/UpdateProfileScreen'
+import UpdateprofileScreen from "./screens/UpdateProfileScreen";
 import ProfileScreen from "./screens/ProfileScreen";
-import UpdatePassword from './screens/UpdatePassword';
-import PhoneNumberLoginScreen from './screens/PhoneNumberLoginScreen';
+import UpdatePassword from "./screens/UpdatePassword";
+import PhoneNumberLoginScreen from "./screens/PhoneNumberLoginScreen";
 import { HomeProvider } from "./context/HomeContext";
 import ClubSearchScreen from "./screens/ClubSearchScreen";
 import ErrorScreen from "./screens/ErrorScreen";
```
Comment (murucbt) [inline backend/frontend/src/App.js:17]: Same comment, arrangement issue.

File: backend/frontend/src/components/Footer.js [hunk 4]
```diff
@@ -1,14 +1,149 @@
...
+      </div>
+
+      <div className="footer-bottom">
+        <div className="container">
+          <div className="copyright">
+            <div className="row align-items-center">
+              <div className="col-md-6">
+                <div className="copyright-text">
+                  <p className="mb-0">
+                    &copy; 2024 STRONGR - All rights reserved.
```
Comment (murucbt) [inline backend/frontend/src/components/Footer.js]: Change the year to dynamic value.

File: backend/frontend/src/components/Testimonial.js [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+import $ from "jquery";
+import OwlCarousel from "react-owl-carousel";
+import "owl.carousel/dist/assets/owl.carousel.css";
+import "owl.carousel/dist/assets/owl.theme.default.css";
+import "../css/testimonial.css";
+import testimonialProfile from "../images/profile.jpg";
+
+// Ensure jQuery is available globally
+window.jQuery = $;
+window.$ = $;
```
Comment (murucbt) [inline backend/frontend/src/components/Testimonial.js]: These two window.* related things are repeated in multiple places, can you try to move it to common place?

File: backend/frontend/src/components/Testimonial.js [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+const Testimonial = () => {
+  const testiMonials = [
+    {
+      title: "Personalized Attention",
+      name: "Ariyan Rusov",
+      description:
+        "DreamSports' coaching services enhanced my badminton skills. Personalized attention from knowledgeable coaches propelled my game to new heights.",
```
Comment (murucbt) [inline backend/frontend/src/components/Testimonial.js]: content has to be changed here and related places (below too).

File: backend/frontend/public/index.html (hunk 2 above)
Comment (rajasekarcbt) [reply backend/frontend/public/index.html]: Yes, I have formatted it in ascending order.

File: backend/frontend/src/App.js (hunk 3 above)
Comment (rajasekarcbt) [reply backend/frontend/src/App.js:17]: yes I formatted the arrangement issue

File: backend/frontend/src/components/Footer.js (hunk 4 above)
Comment (rajasekarcbt) [reply backend/frontend/src/components/Footer.js]: yes I changed the dynamic value

File: backend/frontend/src/components/Testimonial.js (hunk 5 above)
Comment (rajasekarcbt) [reply backend/frontend/src/components/Testimonial.js]: yes, I added those lines to index.js file; it's a common file

File: backend/frontend/src/components/Testimonial.js (hunk 6 above)
Comment (rajasekarcbt) [reply backend/frontend/src/components/Testimonial.js]: Indeed, DreamSports has been renamed as Stronger.

File: backend/frontend/public/assets/plugins/owl-carousel/owl.video.play.html (hunk 1 above)
Comment (rajasekarcbt) [reply backend/frontend/public/assets/plugins/owl-carousel/owl.video.play.html]: Yes, I modified the format.

**Review by rugvedcbt — APPROVED** [review]

**Pull request merged**